modules/installation/pages/Change-Port

```diff
@@ -0,0 +1,36 @@
+= Changing IP or Hostname
+
+If you need to change the IP address or hostname of a node or a cluster after TigerGraph is up and running, 
+you can change them by running the following commands as the tigergraph admin user.(Search for the JSON field called "Hostlist").
```

Comment:
```suggestion
you can change them by running the following commands as the tigergraph admin user.
```
The comment about searching for "Hostname" belongs in step 4 - editing the .tg.cfg file.